src/main/groovy/life/qbic/datamodel/dtos/business/ProductItem.groovy

```diff
@@ -40,6 +40,13 @@ class ProductItem {
      */
     final double quantityDiscount
 
+    /**
+     * Stores the latest position on the offer.
+     *
+     * A negative value indicates no specific position was stored in the item.
+     */
+    private int offerPosition = -1
```

Comment:
why private? the other fields are not set private.

I would do it consistent for all fields. Either private setters or not. And I think there was a reason to not use them but I cant remember